celer/tests/test_docstring_parameters.py

```diff
@@ -108,7 +109,7 @@ def check_parameters_match(func, doc=None):
 # @requires_numpydoc
 def test_docstring_parameters():
```

Comment:
I cannot pass this test either locally or in the test workflow.
In both cases, I get the following error

```bash
celer\tests\test_docstring_parameters.py:111: ModuleNotFoundError
```

Reply:
do you have numpydoc installed ?

Reply:
On the CI, it fails because some docstring parameters are not in the same order as i the function definition: see https://github.com/mathurinm/celer/runs/5746219541?check_suite_focus=true#step:5:208